src/compiler/transformers/es6.ts

```diff
@@ -609,12 +609,14 @@ namespace ts {
             if (node.name) {
                 enableSubstitutionsForBlockScopedBindings();
             }
+            const closingBraceLocation = { pos: node.end - 1, end: node.end };

```

Comment:
In the emitter, the return statement's position is mapped to `node.members.end`. Also, rather than defining this here and passing it down as an argument, couldn't you derive it inside of `transformClassBody` as its the only place that uses it?